kclvm/tools/src/LSP/src/hover.rs

```diff
@@ -55,7 +58,37 @@ pub(crate) fn hover(
                             }
                             docs.insert(attrs.join("\n\n"));
                         }
-                        // todo: hover ScopeObjectKind::Attribute optional, default value
+                        ScopeObjectKind::FunctionCall => {
+                            let ty = obj.ty.clone();
+                            match &ty.kind {
+                                kclvm_sema::ty::TypeKind::Function(func_ty) => {
+                                    // system package function
```

Comment:
Directly use a function to encapsulate the display style of function signatures?